float32 shoulder1_angle
float32 shoulder2_angle
float32 elbow1_angle
float32 elbow2_angle
float32 wrist_angle
float32 endeffector_angle
